src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -32,8 +32,28 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
 
         public static IEnumerable<TexlFunction> FunctionList => FuncsByName.Keys;
 
+        private static readonly object LockObject = new object();
```

Comment:
Is this used?

Reply:
It was in an initial iteration, not anymore. I'll remove it.